core/float/shared/arithmetic_exception_in_coerce.rb

```diff
@@ -0,0 +1,24 @@
+require File.expand_path('../../fixtures/classes', __FILE__)
+
+describe :float_arithmetic_exception_in_coerce, shared: true do
+  ruby_version_is "" ... "2.5" do
```

Comment:
The rest of the codebase does not use spaces around `...` (at least in version guards)